index.bs

```diff
@@ -1057,9 +1056,9 @@ fragment:</p>
 
   <ol algorithm="computed position">
 
-   <li><p>If the <a lt="WebVTT cue position">position</a> is numeric, then return the value of the
-   <a lt="WebVTT cue position">position</a> and abort these steps. (Otherwise, the <a lt="WebVTT cue
-   position">position</a> is the special value <a lt="WebVTT cue automatic
+   <li><p>If the <a lt="WebVTT cue position">position</a> is numeric between 0 and 100, then return
```

Comment:
I think this change is, strictly speaking, unrelated to this bug. If you include it, I suggest:

"... is **a** numeric **value** between 0 and 100 ..."

Reply:
Yes, that's true - it's a fly-by error I noticed as a consequence of checking the margin calculations. I could open another PR, but it's slightly related because it might cause wrong position calculations. If you're ok with it, I'll just fix the language as proposed.

Reply:
Sure, that is fine with me.